<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Invoice</class>
 <widget class="QWidget" name="Invoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>686</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>801</width>
    <height>686</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="adminLoginLabel_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TIckets</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="timingtextEdit_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Movie</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tickettextEdit_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="movietextEdit_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Timing</string>
   </property>
  </widget>
  <widget class="QLabel" name="mainlabal_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 19pt &quot;Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Payments</string>
   </property>
  </widget>
  <widget class="QComboBox" name="methodcomboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debit Card</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Credit Card</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UPI/Paytm</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment Method</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>UPI/Paytm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="paypushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>610</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelpushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>610</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="listMoviecomboBox_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_9">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Seats (Row-Seat )</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Full Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Email Id</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name on Card</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_14">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Card Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_15">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>540</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CVV</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>480</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Card Expiry (MM/YY)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="upiLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>531</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="namecardLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardNoEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>530</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="expLineEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>480</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cvvLineEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>530</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_16">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Ticket Price</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tickettextEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_17">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Discount Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_18">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Amount to be Paid</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="totaltextEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="discoutpushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="discountTextEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adminLoginLabel_19">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Discount Amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="codeLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
